stable/rabbitmq/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 6.8.3
+version: 6.8.4
```

Comment:
I think this should increased the minor section of the version instead of the patch

Reply:
Done